examples/trials/cifar10_pytorch/config.yml

```diff
@@ -1,23 +1,14 @@
-authorName: default
-experimentName: example_pytorch_cifar10
+searchSpaceFile: search_space.json
```

Comment:
suggest at least for this simple example, we add "experimentName", because if users really use nni to run experiments, they want to give each experiment an easy-to-remember name (not experiment ID). If this field is not added in the example, users have to check config references, which is not friendly